website/content/maintainers/governance/governance-self-assessment.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "Governance Self Assessment"
+date: 2023-09-15
+weight: 40
+---
+
+Governance reviews contribute to the health and sustainibility of the CNCF projects. By providing guidance on
+effective governance practices, TAG Contributor Strategy aims to ensure that projects operate efficiently,
+encourage diverse participation, and uphold the values of the CNCF. The governance review process is designed to be
+constructive and supportive, aiming to assist projects in refining their governance models and addressing
+any challenges they may face.
+
+Projects may ask TAG Contributor Strategy for assistance in resolving any issues uncovered by the review. The TAG is
+available via our [slack channel](https://cloud-native.slack.com/archives/CT6CWS1JN),
+[email](https://lists.cncf.io/g/cncf-tag-contributor-strategy),
+[GitHub](https://github.com/cncf/tag-contributor-strategy), or by joining our weekly meetings
+(listed on the [CNCF public calendar](https://www.cncf.io/calendar/)).
+
+
+TODO: LATER: How to request a TAG CS governance review?
+
+## General Instructions
+Instructions:
+- Copy and paste the template in [Self Assessment Template](governance-self-assessment-template.md) into your favorite editor.
+- Fill in the template with the information about your project. The `TODO` placeholders in the template are there to help you.
+- Pick the latest state of the governance document and use the same point-in-time reference (e.g. commit) for your assessment. If governance consists of multiple documents, make sure you use the same commit.
+    - TODO: what about websites?
+- Make sure all links in your report are permanent links (e.g. with commit hashes).
+- To ensure consistency, use the same commits as links for multiple files within a single repository.
+- It is recommended to start filling this template from the "Review" section. Once you fill everything there, go back to "Summary and Assessment" to summarize the content you wrote.
+    - TODO: do we even need a "Summary and Assessment" section for self-assessments?
+
+## Template details
+
+### Project Information
+
+Fill in the blanks in this section.
+
+##### Executing the Assessment
+
+* Write a brief description that details the timebox the assessment occurred and the individuals involved in the assessment.
+* The description should contain the reason for the assessment.
+
+> **_NOTE:_** Make sure you use a snapshot of the governance documents for your assessment and note the commit hash of the snapshot here as a link.
+
+#### Review
+
+##### Governance Description
+
+* Write a narrative describing
+    * governance type of the project
+    * number of substantial contributors and their grouped employer affiliations
```

Comment:
I don't think "substantial" contributors is part of the governance criteria, so this should probably be maintainers?

Reply:
Updated